performance/models.py

```diff
@@ -7,18 +7,19 @@
 
 def eqprs_file_path(instance, filename):
     return (
-        f"eqprs_uploads/{uuid.uuid4()}/{filename}"
+        # f"eqprs_uploads/{uuid.uuid4()}/{filename}"
+        f"eqprs_uploads/{filename}"
     )
 
 
 class EQPRSFileUpload(models.Model):
-    user = models.ForeignKey(User, models.DO_NOTHING)
-    task = models.ForeignKey(Task, on_delete=models.CASCADE)
+    user = models.ForeignKey(User, models.DO_NOTHING, default=0)
```

Comment:
Why do we set a default here? Isnt it sufficient to  guarantee it gets set? Ideally setting it serverside so that user's can't mess with it at all like in the first example here https://books.agiliq.com/projects/django-admin-cookbook/en/latest/current_user.html

Reply:
this is to set the first user in the user dropdown as selected in default. happy to change if you think it's not the ideal way

Reply:
mmm the 0 here refers to the user id in the database, not the position in the dropdown.
Also, this is a bit of a security issue - we don't want users to be able to set it as some other user, so it shouldn't happen client side - it should be set server-side. Please use the example above.

Note we want the creation user
![Screenshot_2023-01-03_14-28-21](https://user-images.githubusercontent.com/235801/210357371-403fa2f7-4d73-452e-ae7f-ece4e1acd217.png)

we **don't** want it to change every time it's edited:
![Screenshot_2023-01-03_14-29-02](https://user-images.githubusercontent.com/235801/210357455-38de288c-61f1-4530-bfec-b3e408bd5cbe.png)

Reply:
if we need to set _some_ value, I think changing to allowing null is better than setting it to zero - we never want to use an actual value when we mean "no value". 0 could be an actual user.

Reply:
I wanted the user dropdown to be filled in default(even before it is saved). that's why I was playing with the `default` field. changed it now so the user is set in the backend